### PR TITLE
Issue #277 chore/styles: cleans up auth request notificaiton to only show details on hover

### PR DESCRIPTION
**Pull request by hunterlester**
@joshuef At first I tried placing the styles in `notifier.css`, where I noticed them not loaded when `Notifier` is rendered. There are absolutely no styles that will take effect from `notifier.css`. What is that about?

I then tried placing styles in `authReqest.css` and importing it in `authRequest.js`, for which I get parsing error related to babel, which I don't understand yet because the config in `webpack.config.base.js` looks just fine.

### CODE REVIEW

**Comment by hunterlester** [comment]
@joshuef The only way I can get the styles to take effect are to place them in `app.global.css`.

File: app/components/Notifier/notifier.css [hunk 1]
```diff
@@ -8,3 +8,13 @@
 
     font-size: 1.4rem;
 }
+
+.hide
```
Comment (joshuef) [inline app/components/Notifier/notifier.css]: If we're going to have a general class for hiding content, I think `hidden` is a better name.

But if we're doing _that _ it's not so normal for it to be shown (yet still have this class... [via some css trickery]).

So I think we should rename these classes. We can be more specific (even if it needs to be in app.global for now).

maybe: `more_details_button` and `more_details_details`? But that could be too specific... `info_box_details` and `info_box_expander` might work and be clear enough... and then `info_box` is something with these features...

File: app/app.global.css [hunk 2]
```diff
@@ -27,3 +27,13 @@ html
     font-size: 62.5%;
     line-height: 1.2em;
 }
+
+.info_box_details
+{
+    display: none;
+}
+
+.info_box_exander:hover .info_box_details
```
Comment (joshuef) [inline app/app.global.css]: `exander` -> `expander`

**Comment by joshuef** [comment]
@hunterlester this is working nicely.

But let's get more space between the details 'link' and the popup: 
https://www.dropbox.com/s/jckohhkogn936r3/Screenshot%202018-07-24%2008.41.31.jpg?dl=0

Also, can make sure that the details link is confined to the text only (ie. that you cant open up details by moving the mouse over the area to the right of the text).

**Pull request merged**